cypress/integration/visitorIsAbletoSeeTopTenMovies.feature.js

```diff
@@ -0,0 +1,38 @@
+describe('Visitor can see top 10 movies', () => {
+	describe('successfully', () => {
+		before(() => {
+			cy.intercept('GET', 'https://worldwidenetflix.herokuapp.com/api/movies', {
+				fixture: 'top10movies.json',
+			});
+		});
+
+		it('is expected to show a list of top 10 global movies', () => {
+			cy.visit('/');
+			cy.get('[data-cy=movie-container]').within(() => {
+				cy.get('[data-cy=movie-0]').within(() => {
+					cy.get('[data-cy=title-header]').should(
+						'contain',
+						'The Shawshank Redemption'
+					);
+				});
+			});
+		});
+	});
```

Comment:
The test expects a list of 10 items, but I don't see an assertion for something being equal to 10!